packages/fluentui/docs/src/index.tsx

```diff
@@ -1,14 +1,9 @@
+import 'react-hot-loader';
```

Comment:
Applying patch, it should be [the first import](https://github.com/gaearon/react-hot-loader#getting-started).